core/src/main/java/org/apache/brooklyn/core/mgmt/ha/OsgiArchiveInstaller.java

```diff
@@ -336,14 +358,14 @@ private synchronized void close() {
             if (!updating) { 
                 osgiManager.managedBundlesRecord.addManagedBundle(result);
                 result.code = OsgiBundleInstallationResult.ResultCode.INSTALLED_NEW_BUNDLE;
-                result.message = "Installed "+result.getMetadata().getVersionedName()+" with ID "+result.getMetadata().getId();
+                result.message = "Installed Brooklyn catalog bundle "+result.getMetadata().getVersionedName()+" with ID "+result.getMetadata().getId()+" ["+result.bundle.getBundleId()+"]";
```

Comment:
this will happen if you install a `catalog.bom` which doesn't come from OSGi and which declare a bundle.  we should be declaring a `bundle` in all BOMs.  i've added this (some cheekily pushed to other projects).

the odd thing though is that the "delete empty BOMs" logic should be cleaning this up.  i'm looking at that.  (i think all other comments are addressed.)